worker/src/handlers/statement/statementForwarding/getStatementsRepo.js

```diff
@@ -0,0 +1,61 @@
+import createStatementsRepo from '@learninglocker/xapi-statements/dist/repo/facade';
+import { getConnection } from 'lib/connections/mongoose';
+import { createClient } from 'lib/connections/redis';
+import once from 'lodash/once';
+import defaultTo from 'lodash/defaultTo';
+import { join as joinPath } from 'path';
+
+export default once(() => {
+  const endpoint = defaultTo(process.env.FS_LOCAL_ENDPOINT, process.cwd());
+  const subFolder = defaultTo(process.env.FS_SUBFOLDER, 'storage');
+  const statementsSubFolder = defaultTo('statements', process.env.SUB_FOLDER_STATEMENTS);
+  return createStatementsRepo({
+    auth: {
+      facade: 'mongo',
+      mongo: {
+        db: getConnection(),
+      },
+    },
+    events: {
+      facade: process.env.EVENTS_REPO,
+      redis: {
+        client: createClient(),
+        prefix: process.env.REDIS_PREFIX,
+      },
+      sentinel: {
+        client: createClient(),
```

Comment:
Will this create a Sentinel client even if its not configured to need it?

Reply:
Because of how that function works it would have just passed in either a Redis or Sentinel client depending on what was needed and not used it. I've updated this code to just pass null connections through since we don't actually need them right now anyway.